atd-vzv/src/views/map/Map.js

```diff
@@ -336,7 +335,7 @@ const Map = () => {
       getCursor={_getCursor}
       interactiveLayerIds={interactiveLayerIds}
       onClick={_onSelectCrashPoint}
-      ref={(ref) => (mapRef.current = ref && ref.getMap())}
+      ref={mapRef}
```

Comment:
This change updates the shape of the ref object. We were assigning the map instance to `ref.current` directly by calling [getMap()](https://github.com/visgl/react-map-gl/blob/6.1-release/docs/api-reference/static-map.md#getmap) on assignment. The geocoder added in this PR is[ expecting to call `getMap()` on the ref passed to it](https://github.com/SamSamskies/react-map-gl-geocoder/blob/2968c74e730c52f2ecece086e08ed9d518fd9bfd/src/index.js#L119-L123) and was breaking when trying to call map methods in conjunction with the custom way this ref was assigned.

**Before**
`ref.current` => the Mapbox map instance

**Now**
`ref.current.getMap()` => the Mapbox map instance

I updated the other uses of the map instance to get it with `getMap()` so they would continue to work.